compose/app.yaml

```diff
@@ -58,5 +55,23 @@ services:
       - "8002"
     labels:
       traefik.enable: true
-      traefik.frontend.rule: 'Host: api.wb.local; PathPrefix:/trading, /trading-ui-assets'
+      traefik.frontend.rule: 'Host: www.wb.local; PathPrefix:/peatio/trading, /peatio/trading-ui-assets'
       traefik.port: 8002
+
+  assets:
+    image: nginx
+    depends_on:
+      - peatio
+      - barong
+      - proxy
```

Comment:
I think that peatio depends on it's assets ?
I'm not sure just asking

Reply:
start `peatio` container -> put assets into the `peatio-assets` volume -> serve this assets by nginx.

Reply:
peatio-assets will run correctly without peatio but peatio will not